src/Surfnet/StepupMiddleware/ApiBundle/Response/JsonCollectionResponse.php

```diff
@@ -51,6 +54,10 @@ public function __construct($totalItems, $page, $pageSize, array $collection, $h
             'items'      => $collection
         );
 
+        if (!empty($filters)) {
+            $data['filters'] = $filters;
```

Comment:
filters might not be set on the response data. This might cause problems in RA.

Reply:
Agreed, please don't forget the update of the API docs!

Reply:
Fixed this to always set data